### PR TITLE
PRIME-676 - Business Licence Upload Frontend

### PR DESCRIPTION
**Pull request by AnaisHebert**
Blocked by https://bcgovmoh.atlassian.net/browse/PRIME-695

### CODE REVIEW

File: openshift/document-manager.bc.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+          - name: DB_NAME
+            valueFrom:
+              secretKeyRef:
+                key: database-name
+                name: "postgresql-secret"
+          - name: CACHE_REDIS_PASS
+            valueFrom:
+              secretKeyRef:
+                key: database-admin-password
+                name: "postgresql-secret"
```
Comment (james-hollinger) [inline openshift/document-manager.bc.yaml:98]: Is it intentional that the redis cache has the same password as the postgres database?

Comment (hogmann) [reply openshift/document-manager.bc.yaml]: @poggdogg

**Comment by poggdogg** [comment]
Pulled out my templates - this is good for now.  I will build templates in another PR

File: openshift/document-manager.bc.yaml (hunk 1 above)
Comment (james-hollinger) [reply openshift/document-manager.bc.yaml]: After some discussion, it seems so.

**Pull request merged**